### DISCUSSION TITLE
Can't get plugin to work

### DISCUSSION THREAD

**Discussion by BlueRocketMouse** [ankenyr/jellyfin-youtube-metadata-plugin#41] [Q&A] [answered]
I followed the directions for installing from the repository and installed the latest version available in the catalogue, version 1.0.3.3. The plugin showed up as active and all looked good, but YouTubeMetadata did not appear as an option in any of my fetchers for my library. I then uninstalled the plugin and attempted to manually install version 1.0.3.5 instead, but could not get that to work either (I just kept getting "Status: Malfunctioned").

What am I doing wrong here? My server is version 10.7.7 and I am using a "Shows" library for this.

**Comment by ankenyr** [comment]
What libraries were you using? Tv? Movies?
The latest only works with 10.8 or higher so it makes sense that it didn't
work.

**Reply by BlueRocketMouse** [reply]
Gotcha, so it sounds like 1.0.3.3 is indeed what I should be using then. I'm using a Shows/TV library for all my YouTube content.

**Comment by ankenyr** [comment, marked as answer]
You will need to move to 10.8 then as my plugin did not support TV for 10.7. Certain features are missing in Jellyfin that prevented me from using TV.

**Reply by BlueRocketMouse** [reply]
Good to know. Thanks for the help, I'll give the plugin another try once 10.8 fully releases.

**Comment by ankenyr** [comment]
I have been using it for a while with no big issues so far.

**Comment by ilkersigirci** [comment]
I am using latest version 10.8 linuxserver/jellyfin:nightly with your plugin version with 1.3.5. It succesfully installed but I got an error when clicking plugin settings.

```
[21:16:47] [WRN] [24] Microsoft.EntityFrameworkCore.Model.Validation: The entity type <TYPE> is configured to use schema 'jellyfin', but SQLite does not support schemas. This configuration will be ignored by the SQLite provider
```

- Type fields are the followings:
  - AccessSchedule
  - ActivityLog
  - CustomItemDisplayPreferences
  - DisplayPreferences
  - HomeSection
  - ImageInfo
  - ItemDisplayPreferences
  - Permission
  - Preference
  - ApiKey
  - Device
  - DeviceOptions
  - User

**Comment by ankenyr** [comment]
Not sure why that is coming up. There should be no more config

https://github.com/ankenyr/jellyfin-youtube-metadata-plugin/blob/master/Jellyfin.Plugin.YoutubeMetadata/Configuration/PluginConfiguration.cs

Maybe the existence from the previous version makes jellyfin want to load
something but that fails? Either way you are not missing anything. You
could go into your files and probably remove it. You may want to make a
backup first before you do that just in case.

**Reply by ilkersigirci** [reply]
Actually, I freshly installed 10.8 using docker, so there shouldn't be any old config. Then, I reinstalled everything again to be sure. Unfortunately, the same problem occurred. Maybe it is linuxserver image problem. I will dig into it. Thanks

**Reply by ilkersigirci** [reply]
I switched to jellyfin/jellyfin docker image instead of linuxserver one. I tested again jellyfin 10.8 alpha 5 with your plugin version 1.3.5. When I tried to access youtube-dl plugin settings page, I got blank page again. What am I doing wrong? Other plugins like traktv are working fine and I can succesfully access their settings page.

**Reply by ankenyr** [reply]
The settings page should be blank. There are no settings for this version. It no longer uses the Google API so there is no need for the Google API key

**Reply by ilkersigirci** [reply]
Ok, understand it now. Still my thumbnails not showing, but I will figure it out. Thanks

**Reply by ankenyr** [reply]
Got a screenshot of what you are seeing (or not seeing) also can you provide me the directory structure for those files from the root of the library? You can replace the ID and file name with anything if you don't wish to publicize the content itself. Also logs could be helpful.

**Reply by ilkersigirci** [reply]
I attached the SS you requested in below. The library type is Movie as if I select Shows, it doesn't show anything at all.  As you can see from the library SS, metadata is duplicated and the thumbnail is set the same for every video. My other libraries of movies and tv shows doesn't have these problems, hence I believe it is the plugin problem. I hope you can assist me to solve the problem.

NOTE: I tried to use videos with/without info.json files. As expected, videos without info.json couldn't provide metadata but with info.json , metadata is just the same and duplicated across the library.

![Screenshot (36)](https://user-images.githubusercontent.com/39417966/156932321-8d2cadb7-041f-4b36-9035-9393375dbbd1.png)
![Screenshot (37)](https://user-images.githubusercontent.com/39417966/156932328-d1b2741f-45f0-4f4e-a363-2615b7bd0c84.png)
![Screenshot (33)](https://user-images.githubusercontent.com/39417966/156932330-c05e9e83-74d8-4379-a204-9febf956532d.png)
![Screenshot (34)](https://user-images.githubusercontent.com/39417966/156932334-caa3bd6a-2bce-49f5-b670-0769e0e847bf.png)
![Screenshot (35)](https://user-images.githubusercontent.com/39417966/156932338-773c41d2-0001-49fb-9288-fdc3925642fa.png)

**Reply by ankenyr** [reply]
Is the view of each file something within Jellyfin? I don't believe I have seen that. How do you navigate to that?

**Reply by ilkersigirci** [reply]
It is a SS from an app called [filebrowser](https://github.com/filebrowser/filebrowser). All my youtube videos in the same folder which i imported to Jellyfin. Idk what is the problem here?

**Reply by ankenyr** [reply]
>Ok, understand it now. Still my thumbnails not showing, but I will figure it out. Thanks

I thought this was the problem? So far I don't see a problem in the screenshots though.

**Reply by ilkersigirci** [reply]
In the fourth screenshot, the thumbnail images are all the same even though, each video has a respected webp file. So I expected to have them to show in the jellyfin library.

**Reply by ankenyr** [reply]
Ok so if I understand this correctly. Your directory consists of only the files shown in the filebrowser screenshot. However when you go to your movies, it seems that they were all renamed to the one video and use that videos screenshot?

**Reply by ilkersigirci** [reply]
Exactly

**Reply by ankenyr** [reply]
I would recommend turning off the other meta data agents and doing a forcefull refresh of the metadata (remember to click the 
![image](https://user-images.githubusercontent.com/1542958/157093712-30daa488-0c18-412c-8a7b-5882720aec46.png)

If that doesn't work can you turn on debug logs and provide me with those logs?

**Reply by ilkersigirci** [reply]
I have already tried that with no luck. In the docker logs there doesn't  seems to be any error.

**Comment by ankenyr** [comment]
The logs may not be explicitly of type "Error" if you turn on debugging for
logging there are logs that can help determine what is happening and
decisions that are being made.
https://github.com/ankenyr/jellyfin-youtube-metadata-plugin/blob/e56077bfabbe11d62c2ea309518e7ef9f0ea95c3/Jellyfin.Plugin.YoutubeMetadata/Providers/LocalMetadata/YoutubeLocalSeasonProvider.cs#L22
That log line is not an error but it could help me. I will also be pushing
a new version soon. If you are on 10.8 I would also recommend using Shows
rather than Movie if that fits how you would want to view the content.
